solidity/contracts/isms/TrustedRelayerIsm.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+// ============ Internal Imports ============
+import {IInterchainSecurityModule} from "../interfaces/IInterchainSecurityModule.sol";
+import {Message} from "../libs/Message.sol";
+import {Mailbox} from "../Mailbox.sol";
+
+contract TrustedRelayerIsm is IInterchainSecurityModule {
+    using Message for bytes;
+
+    uint8 public moduleType = uint8(Types.NULL);
+    Mailbox public immutable mailbox;
+    address public immutable trustedRelayer;
```

Comment:
update `deriveNullConfig` in the ISM reader pls, if contract has `trustedRelayer` property then you can infer it's a `TrustedRelayerIsm`